doc/source/lilac/obtaining-building-and-running/obtaining-and-building-ctsm.rst

```diff
@@ -145,13 +145,13 @@ the `CIME documentation`_.
 Building on a CIME-supported machine
 ------------------------------------
 
-If you are using a machine that has been ported to CIME_ (for example, NCAR's ``cheyenne``
+If you are using a machine that has been ported to CIME_ (for example, NCAR's ``derecho``
 machine), then you do not need to specify much information to ``build_ctsm``. In addition,
 in this case, CIME will load the appropriate modules and set the appropriate environment
 variables at build time, so you do not need to do anything to set up your environment
 ahead of time. **Building CTSM with LILAC requires ESMF. ESMF is currently an optional
```

Comment:
ESMF is not optional for cesm3.  But we are not going to supply it or build it for users - refer them to the esmf.org website for download and build instructions.

Reply:
@jedwards4b thank you for your comments, I found them very helpful.